proto/google/events/cloud/build/v1/data.proto

```diff
@@ -0,0 +1,631 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package google.events.cloud.build.v1;
+
+option csharp_namespace = "Google.Events.Protobuf.Cloud.Build.V1";
+import "google/api/field_behavior.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+// Build event data
+// Common build format for Google Cloud Platform API operations.
+// Copied from
+// https://github.com/googleapis/googleapis/blob/master/google/devtools/cloudbuild/v1/cloudbuild.proto,
+message BuildEventData {
+  // Possible status of a build or build step.
+  enum Status {
+    // Status of the build is unknown.
+    STATUS_UNKNOWN = 0;
+
+    // Build or step is queued; work has not yet begun.
+    QUEUED = 1;
+
+    // Build or step is being executed.
+    WORKING = 2;
+
+    // Build or step finished successfully.
+    SUCCESS = 3;
+
+    // Build or step failed to complete successfully.
+    FAILURE = 4;
+
+    // Build or step failed due to an internal cause.
+    INTERNAL_ERROR = 5;
+
+    // Build or step took longer than was allowed.
+    TIMEOUT = 6;
+
+    // Build or step was canceled by a user.
+    CANCELLED = 7;
+
+    // Build was enqueued for longer than the value of `queue_ttl`.
+    EXPIRED = 9;
+  }
+
+  // Output only. Unique identifier of the build.
```

Comment:
Drive-by review as I stop for the night - it's worth going through all the docs thinking about this as "this is event data; it's only ever consumed". So "output only" and defaults don't really mean very much in that context.

But we can tweak all of that later, of course.

Reply:
+1. I don't think we need the OUTPUT_ONLY option.